content/index.html

```diff
@@ -36,31 +36,32 @@ <h3>LXD</h3>
           <p>LXD is a next generation system container and virtual machine
             manager. It offers a unified user experience around full Linux
             systems running inside containers or virtual machines.</p>
-          
+
           <p>LXD is image based and provides images for a wide number of
             Linux distributions. It provides flexibility and scalability
             for various use cases, with support for different storage
             backends and network types and the option to install on
             hardware ranging from an individual laptop or cloud instance
             to a full server rack.</p>
-          
+
           <p>When using LXD, you can manage your instances (containers and VMs)
-            with a single command line tool, or over the network in a transparent way
-            through a REST API.</p>
+            with a simple command line tool, directly through the REST API or
+            by using third-party tools and integrations. LXD implements a
+            single REST API for both local and remote access.</p>
```

Comment:
:thinking: I would write that the command line tool and the REST API can be used both locally and remotely.